luigi/scheduler.py

```diff
@@ -634,25 +687,35 @@ def add_task(self, task_id=None, status=PENDING, runnable=True,
         if not task.params:
             task.params = _get_default(params, {})
 
+        if batch_id is not None:
+            task.batch_id = batch_id
+        if status == RUNNING and not task.worker_running:
+            task.worker_running = worker_id
+
         if tracking_url is not None or task.status != RUNNING:
             task.tracking_url = tracking_url
+            for batch_task in self._state.get_batch_running_tasks(task.batch_id):
+                batch_task.tracking_url = tracking_url
+
+        if batchable is not None:
```

Comment:
Can we avoid tribool logic here? Like now we have 3 possible values (`None`, `True`, `False`). Could the default value for `batchable` be set to `False`?